internal/executor/exec.go

```diff
@@ -11,15 +11,19 @@ import (
 // The cmdOutput argument is used to print the command output.
 func Run(cmd []string, cmdOutput, cmdError bool, wg *sync.WaitGroup, outputCh, errorCh chan<- string) {
 	defer func() {
+		close(outputCh)
+		close(errorCh)
 		wg.Done()
 	}()
 
 	command := exec.Command(cmd[0], cmd[1:]...)
 	stdout, _ := command.StdoutPipe()
 	stderr, _ := command.StderrPipe()
 
-	//command.Wait()
-	command.Start()
+	if err := command.Start(); err != nil {
+		fmt.Printf("command execution error: %v\n", err)
```

Comment:
can you send it through `errorCh` instead of printing it?
```
errorCh <- fmt.Sprintf("command execution error: %v", err)
```
Please change the test accordingly.
This should look as follow:
```
stderr 'error: command execution error: ...'
```

Reply:
@alegrey91 sure, I've explained [here](https://github.com/alegrey91/harpoon/pull/60#discussion_r1894672121) why I did like that. 
Let me know if I should proceed in using the already existing `errorCh` :smile: